frontend/public/components/modals/create-namespace-modal.jsx

```diff
@@ -129,26 +130,48 @@ const CreateNamespaceModalWithTranslation = connect(
         return (
           <>
             <p>{nameFormat}</p>
-            {this.props.createProject ? <p>{createNamespace}</p> : null}
+            {createProject ? <p>{createNamespace}</p> : null}
           </>
         );
       };
+
       return (
         <form
           onSubmit={this._submit.bind(this)}
           name="form"
           className="modal-content modal-content--no-inner-scroll"
         >
           <ModalTitle>
-            {this.props.createProject ? t('public~Create Project') : t('public~Create Namespace')}
+            {createProject ? t('public~Create Project') : t('public~Create Namespace')}
           </ModalTitle>
           <ModalBody>
+            {createProject ? (
+              <>
+                <p>
+                  {t(
+                    'public~An OpenShift project is an alternative representation of a Kubernetes namespace.',
+                  )}
+                </p>
+                <p>
+                  <ExternalLink
+                    href={`${openshiftHelpBase}applications/projects/working-with-projects.html`}
+                  >
+                    {t('public~Learn more about working with projects')}
+                  </ExternalLink>
+                </p>
+              </>
+            ) : null}
+
             <div className="form-group">
               <label htmlFor="input-name" className="control-label co-required">
                 {t('public~Name')}
               </label>{' '}
               <Popover aria-label="Naming information" bodyContent={popoverText}>
-                <Button variant="plain" aria-label="View naming information">
+                <Button
+                  className="co-button-help-icon"
+                  variant="plain"
+                  aria-label="View naming information"
```

Comment:
Know this was already there, but wondering if aria-labels should also be translated.  This text is read by screen reader users when they hit this object. 

@rebeccaalpert Wondering if we have an existing pattern for aria-label text?

Reply:
Yes! We should translate them. The tags to worry about are all listed in our shiny new i18n documentation: https://github.com/openshift/console/blob/master/INTERNATIONALIZATION.md.